assets/js/translator.js

```diff
@@ -1111,14 +1114,71 @@ function setRecentDstLangs(langs /*: string[] */) {
     return langs;
 }
 
+function dictionaryLookup() {
+    if(modeEnabled('lookup')) {
+        if(currentLookupRequest) {
+            currentLookupRequest.abort();
+        }
+
+        $('#dictionaryLookupResult').empty();
+        var wordToLookup = $('#originalText').val().trim().split(' '); // eslint-disable-line newline-per-chained-call
+        if(wordToLookup.length <= 3 && wordToLookup[0] !== '') {
```

Comment:
Recreating comment about where to draw the line between dictionary translations and full text translations, and also how to handle multiwords.